### DISCUSSION TITLE
Nuclei Windows.exe---smtp.sendgrid.net

### DISCUSSION THREAD

**Discussion by ghost** [projectdiscovery/nuclei#2869] [Q&A] [answered]
Hi I would like ask why Windows Nuclei version exe need use this connection .?

---------------------------
smtp.eu-west-1.mandrillapp.com
34.241.27.172
34.253.241.177
54.78.186.219
34.255.97.37 
--------------------------
smtp.mandrillapp.com
18.208.190.224
3.80.109.92
3.91.68.97
54.209.22.5
54.226.47.131
54.236.7.116 
------------------------
smtp.sendgrid.net
107.20.8.136
167.89.115.116
167.89.115.117
167.89.115.145
167.89.115.18
167.89.115.53
167.89.115.79
167.89.123.149
167.89.123.53
167.89.123.58
167.89.123.82
167.89.123.95
167.89.123.97
34.237.250.201
52.0.142.242
52.204.68.213
54.146.218.5
54.157.14.6 
-------------------
smtp.us-east-1.mandrillapp.com
18.208.190.224
3.80.109.92
3.91.68.97
54.209.22.5
54.226.47.131
54.236.7.116

**Comment by ehsandeep** [comment, marked as answer]
@kolor33, there is a nuclei template for token validation against mandrillapp platform -

https://github.com/projectdiscovery/nuclei-templates/blob/a78a478773680f054bfe5819f207f567188d3eac/token-spray/api-mailchimp.yaml#L17

ideally, this template runs when token value is provided as CLI variable, also what version of nuclei you are running?
